python/datashaper/datashaper/engine/verbs/__init__.py

```diff
@@ -1,2 +1,38 @@
+from . import strings  # noqa F401, F403
```

Comment:
Wondering if there could be a way to avoid large inits with that many imports, this also happens on the pipeline

Reply:
`__init__.py` file is to expose the public API of a module, in recent versions of python (3.3+) you can just skip it if you don't want to define a public API for that module... In that case you just need to use the full import path (up to the file you need instead of the module). In this particular case we need to explicitly import the verbs so that the @verb decorator is executed, otherwise we would need to import all the verbs somewhere else.

Reply:
Could this be done lazily? Like, the verb registration is done on first lookup?